src/plugins/wmr/client.js

```diff
@@ -114,11 +124,19 @@ function getMod(url) {
 	return mod;
 }
 
+let queue = [];
```

Comment:
I think this should be defined in line 9, so it gets defined before use, even though that might not matter here